_glaredb/sql-commands/create-credentials.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: CREATE CREDENTIALS
+parent: SQL commands
+---
+
+# CREATE CREDENTIALS
+
+Create credentials for accessing AWS and Google Cloud resources.
+
+## Syntax
+
+```sql
+CREATE CREDENTIALS <credential-name>
+    PROVIDER <provider>
+    OPTIONS (<provider-options>);
+```
+
+| Field              | Description                                                       |
+| ------------------ | ----------------------------------------------------------------- |
+| `credential-name`  | Name for the credentials.                                         |
+| `provider`         | The cloud provider for these credentials are for. `aws` or `gcp`. |
+| `provider-options` | Provider specific options. See below sections.                    |
+
+## AWS credentials
+
+AWS credentials allows GlareDB to read and write objects in S3.
+
+Creating AWS credentials requires `access_key_id` and `secret_access_key`. These
+correspond to an IAM user with permissions for accessing objects in S3.
+
+```sql
+CREATE CREDENTIALS my_aws_creds
```

Comment:
This would be useful when running locally, but could be confusing when running with hybrid exec. It _would_ be interesting if we could forward `.aws/config` credentials automatically though...